packages/jmix-front-generator/src/building-blocks/stages/writing/pieces/blank-component/blank-component.ts

```diff
@@ -0,0 +1,12 @@
+import {YeomanGenerator} from "../../../../YeomanGenerator";
+
+export function writeBlankComponent<M extends {className: string}>(
```

Comment:
Code is the same as in `writeCardsComponent`. May be it make sence to make default method `writeComponent(name)`

Reply:
I thikn it's ok